### PR TITLE
Convert keypad movement events to equivalent non-keypad ones.

### PR DESCRIPTION
**Pull request by eliotb**
Improves experience for users that only have numpad/keypad movement keys, lacking standalone versions of same.

### CODE REVIEW

File: thonny/tktextext.py [hunk 1]
```diff
@@ -249,6 +250,23 @@ def dispatch(event):
...
+            '<KP_Home>': '<Home>',
+            '<KP_End>': '<End>',
+            '<KP_Next>': '<Next>',
+            '<KP_Prior>': '<Prior>',
+            '<KP_Enter>': '<Return>',
+        }
+        for i in kmap:
+            def mfunc(event, key=i):
+                self.event_generate(kmap[key], **{'state': event.state})
+            self.bind(i, mfunc)
```
Comment (aivarannamaa) [inline thonny/tktextext.py]: In Windows it gives me `_tkinter.TclError: bad event type or keysym "KP_Left"`. Numpad left keycode is just `Left` in Windows. You should catch and silence the binding errors.

I'm also worried that on some platforms this approach may generate double movement actions (one for KP_Left and another for generated Left). The standard approach is ending the event handler with `return "break"` -- this should stop the event propagation.

Comment (aivarannamaa) [reply thonny/tktextext.py]: Also, please add link to the issue as comment inside `bind_keypad`!

**Comment by aivarannamaa** [comment]
Thank you for the contribution! Please see my comments in the code!

**Comment by aivarannamaa** [comment]
BTW. Do you have IDLE installed? Does your numpad movement keys work there?

Which Tk version do you have? ("Thonny => Help => About" and "IDLE => Help => About IDLE")

**Comment by eliotb** [comment]
> BTW. Do you have IDLE installed? Does your numpad movement keys work there?

Just installed, and no, numpad movement doesn't work there either.

> Which Tk version do you have? ("Thonny => Help => About" and "IDLE => Help => About IDLE")

Python 3.7.3, Tk 8.6.9

File: thonny/tktextext.py (hunk 1 above)
Comment (eliotb) [reply thonny/tktextext.py]: I created a new pull request rebased and squashed onto current master branch.

**Comment by aivarannamaa** [comment]
Thanks! I'll test it in my Ubuntu at work.

**Pull request merged**